test/test_sefd.py

```diff
@@ -106,16 +106,17 @@ def test_no_band(poly_model_file) -> None:
         (None, None)
     ]
 )
-def test_to_file(poly_model, antenna: Optional[str], receiver: Optional[str]) -> None:
+def test_to_file(poly_model: sefd.SEFDPoly, antenna: Optional[str], receiver: Optional[str])\
```

Comment:
If you had to split the line, I would do it within the parameter list so that the parentheses of the function declaration provide a natural line extension. There is a general disdain for `\` in Python these days, so folks tend to do what they can to avoid it. From [PEP 8](https://www.python.org/dev/peps/pep-0008/):

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

Reply:
Thanks, agreed (but going to blame PyCharm's auto-wrap behaviour for this one : )

Reply:
Ah yes, that can be sneaky 😄